draft-ietf-rtcweb-security-arch.xml

```diff
@@ -556,6 +556,163 @@
       </section>
     </section>
 
+    <section title="SDP Identity Attribute" anchor="sec.sdp-id-attr">
+      <t>
+	    The SDP 'identity' attribute is a session-level attribute that is used
+	    by an endpoint to convey its identity assertion to its peer. The identity
+        assertion value is encoded as a <xref target="RFC4648">Base-64 string</xref>
```

Comment:
I don't think Base-64 is the best way to describe this. There are three types of base 64 so you need to say which one (and none of them are strings. I would say ' encoded using the base64 algorithm defined in <xref target="RFC4648"/> '

Reply:
Suggest: "Which is encoded as Base-64 as described in section 4 of [RFC4648]."